executable/mach_o.ksy

```diff
@@ -730,6 +754,72 @@ types:
       - id: release
         type: u1
     -webide-representation: '{major:dec}.{minor:dec}'
+  encryption_info_command:
+    seq:
+      - id: cryptoff
+        type: u4
+      - id: cryptsize
+        type: u4
+      - id: cryptid
+        type: u4
+      - id: pad
+        type: u4
+        if: _root.magic == magic_type::macho_be_x64 or _root.magic == magic_type::macho_le_x64
+  twolevel_hints_command:
+    seq:
+      - id: offset
+        type: u4
+      - id: nhints
+        type: u4
+  linker_option_command:
+    seq:
+      - id: count
+        type: u4
```

Comment:
First of all, note that this "style guide" is currently in development (see https://github.com/kaitai-io/kaitai_struct/issues/140 for details), so it's not exactly mandatory to apply every principle.

For naming, it's more like a common sense question. If that's a very common name that everyone knows and it is used in many different implementations, etc. — it probably a good idea to leave it as is (although note that you can't have IDs with upper case letters and various special characters anyway).

In most other cases, usually it's a good idea to match suggested style with "num_", "len_", "ofs_" prefixes, and keep original names using `-orig-id` for search purposes. For example, even if there is some particular naming convention used in some original implementation, usually it's better to cater for KS users (who would be familiar with KS convention), not users of that original implementation.